input/docs/getting-started/index.md

```diff
@@ -21,14 +21,27 @@ Install-Package ReactiveUI.WPF
 ```
 The complete list containing NuGet packages for all supported platforms <a href="https://reactiveui.net/docs/getting-started/installation/nuget-packages/">can be found here</a>. <a href="https://www.nuget.org/packages/reactiveui/">ReactiveUI</a> main package should normally be installed into you .NET Standard class libraries containing platform-agnostic code (repositories, services, DTOs, view-models), and ReactiveUI.XXX packages are platform-specific, so we use <a href="https://www.nuget.org/packages/ReactiveUI.WPF/">ReactiveUI.WPF</a> in this tutorial as we are developing a tiny WPF application that doesn't need code sharing.
 ```
-Install-Package Octokit
+Install-Package Nuget.Client
+Install-Package NuGet.Protocol.Core.v3
 ```
-We also need a Github client library in this tutorial, and we are going to install and use <a href="https://github.com/octokit/octokit.net">Octokit.NET</a>.
+We also need a Nuget client library in this tutorial, and we are going to install and use <a href="https://docs.microsoft.com/en-us/nuget/reference/nuget-client-sdk">NuGet Client</a>.
 
 ### 3. Create a new AppViewModel class
 ```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Windows;
+using NuGet.Configuration;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
```

Comment:
Is it worth showing up usings here? 
Visual Studio (especially with ReSharper) can auto-import them for us.